test/fuse/README.md

```diff
@@ -1,65 +1,115 @@
 # gVisor FUSE Test Suite
 
-This is an integration test suite for fuse(4) filesystem. It runs under both
-gVisor and Linux, and ensures compatibility between the two. This test suite is
-based on system calls test.
+This is an integration test suite for fuse(4) filesystem. It runs under gVisor
+sandbox container with VFS2 and FUSE function enabled.
 
-This document describes the framework of fuse integration test and the
-guidelines that should be followed when adding new fuse tests.
+This document describes the framework of FUSE integration test and the
+guidelines that should be followed when adding new FUSE tests.
```

Comment:
What do you mean by "how to use it"?

Reply:
Can replace "and the guidelines that should be followed when adding new FUSE tests" with "how to use it" to make it concise and compelling

Reply:
I still preserve "guidelines" because this is not just a document describing how to use, but a collection of rules that every developer should follow to develop their own testing features.